os/board/rtl8721csm/src/libs/RELEASE_NOTE.txt

```diff
@@ -1,5 +1,8 @@
 /* == "version" + "Realtek git version" + "compile date" + "compile time" == */
 
+== version a16b340d3cd7a67dd5b27204b60af31557b0d0d8_2022/05/25-18:50:28 ==‎
+1.	Disable WiFi CFO tracking when BT enable to fix BT frequency offset in lib_wlan

```

Comment:
May be we cam modify as below;
1. Disable WiFi CFO tracking when BT is enabled to fix BT frequency offset in lib_wlan